src/types/template.paths.ts

```diff
@@ -0,0 +1,3 @@
+export enum Templates {
```

Comment:
The `service-offline` template should be added on config file here https://github.com/companieshouse/overseas-entities-web/blob/eb7535674ad3a1457b3c0275021bdb4bfd3ccd46/src/config/index.ts#L22

Reply:
Moved templates to config and removed the types folder from this PR.